cmsController.php

```diff
@@ -0,0 +1,195 @@
+<?php
+$db = new PDO('mysql:host=127.0.0.1; dbname=portfolioKyam', 'root');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+/**
+ * Gets about sections name, content from Db and turns into an array
+ *
+ * @param $db PDO to select from
+ * @return assoc array of about sections and content
```

Comment:
dont need to specify assoc

Reply:
resolved.

Reply:
something makes me not want to believe you